### PR TITLE
A small bit of cleanup for the new "remaining time" code

### PR DESCRIPTION
**Pull request by rreed**
This was pushed straight to master with no PR (the branch is now protected) so I'm just doing the small things that I would have asked for here anyway.

PR assigned to an atypical set of people because I need the JS speakers. :)

### CODE REVIEW

File: lib/formatter.js [hunk 1]
```diff
@@ -32,19 +32,20 @@ function dateAsPSTString(date) {
   return moment.tz(date, "America/Los_Angeles").format('YYYY-MM-DD hh:mma z');
 }
 
-function numberPad(n, width) {
-  n = n + '';
-  return n.length >= width ? n : new Array(width - n.length + 1).join('0') + n;
+function leftPad(s, width, padChar = '0') {
```
Comment (thurstonzhu) [inline lib/formatter.js:35]: the famous `leftPad`

**Review by thurstonzhu — APPROVED** [review]
I speak JS as well as I speak Korean, but this _looks_ fine to me

File: lib/formatter.js (hunk 1 above)
Comment (rreed) [reply lib/formatter.js:35]: Should have just installed a new dependency. 🙃

Comment (thurstonzhu) [reply lib/formatter.js:35]: pls no

**Pull request merged**